libs/utils/wa_results_collector.py

```diff
@@ -113,10 +113,12 @@ class WaResultsCollector(object):
     """
     def __init__(self, base_dir=None, wa_dirs=".*", platform=None,
                  kernel_repo_path=None, parse_traces=True,
-                 use_cached_trace_metrics=True):
+                 use_cached_trace_metrics=True, charts=True):
```

Comment:
`display_charts` would be a more fitting name

Reply:
ok